index.html

```diff
@@ -3289,6 +3290,40 @@ <h2 id="resid_misc">Resource identifiers</h2><!-- originally this section has a
 		<p class="advisement">Specifications that define new syntax for URIs or contained within URIs MUST specify that characters outside the ASCII repertoire are <a>percent encoded</a> using the UTF-8 <a>character encoding</a>.</p>
 	</div>
 
+	<section id="idn" class="subtopic">
+		<h3>Domain names</h3>
+		
+		<aside class="links" id="links_lang_decl">
+		<p class="links_title">Useful background and overviews for this section</p>
+		<ul>
+		<li class="w3"><p class="link"><a href="https://www.w3.org/International/articles/idn-and-iri/index.en.html">An Introduction to Multilingual Web Addresses</a></p></li>
+		<li class="w3"><p class="link"><a href="https://www.rfc-editor.org/info/rfc4690">Review and Recommendations for Internationalized Domain Names (IDNs)</a> [[RFC4690]]</p></li>
+		<li class="w3"><p class="link"><a href="https://www.rfc-editor.org/info/rfc5890">Internationalized Domain Names for Applications (IDNA): Definitions and Document Framework</a> [[RFC5890]]</p></li>
+		<li class="w3"><p class="link"><a href="https://www.rfc-editor.org/info/rfc5891">Internationalized Domain Names in Applications (IDNA): Protocol</a> [[RFC5891]]</p></li>
+		<li class="w3"><p class="link"><a href="https://www.rfc-editor.org/info/rfc5892">The Unicode Code Points and Internationalized Domain Names for Applications (IDNA)</a> [[RFC5892]]</p></li>
+		<li class="w3"><p class="link"><a href="https://www.rfc-editor.org/info/rfc5893">Right-to-Left Scripts for Internationalized Domain Names for Applications (IDNA)</a> [[RFC5893]]</p></li>
+		<li class="w3"><p class="link"><a href="https://unicode.org/reports/tr46/">UTS #46: Unicode IDNA Compatibility Processing</a> [[UTS46]]</p></li>
+		</ul>
+		</aside>
+
+		<div class="req" id="idna2003">
+			<p class="advisement">Specificatioins should not refer to RFC 3490, 3491, or 3492 (aka "IDNA2003").</p>
+		</div>
+
+		<p>
+			Comparing with IDNA2003, IDNA2008 (from [[RFC5890]] to [[RFC5893]]) has reduced or eliminated ambiguity, mapping problems, and other issues.
```

Comment:
Ed: suggested reordering:
IDNA2008 (from [[RFC5890]] to [[RFC5893]]) improves on IDNA2003 by reducing or eliminating ambiguity, mapping problems, and other issues.

Reply:
Fixed.